src/installer/pkg/projects/Microsoft.DotNet.ILCompiler.LLVM/ILCompilerRIDs.props

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <!-- The list of RIDs supported by ILCompiler -->
+  <ItemGroup>
+    <OfficialBuildRID Include="linux-arm64" Platform="arm64" />
+    <OfficialBuildRID Include="linux-musl-x64" Platform="x64" />
+    <OfficialBuildRID Include="linux-x64" Platform="x64" />
+    <OfficialBuildRID Include="osx-x64" Platform="x64" />
+    <OfficialBuildRID Include="win-arm64" Platform="arm64" />
+    <OfficialBuildRID Include="win-x64" Platform="x64" />
+    <OfficialBuildRID Include="browser-wasm" Platform="wasm" />
```

Comment:
I guess this one is required to produce the actual WASM runtime (one won't be running ILC on WASM)?

Would be worth a comment to that effect (`This one is a little special because...`).

Reply:
Yes, that's right, its the cross architecture publish flow I suppose you call it.  Is it special, it's the same as compiling for linux on Windows isn't it?

Reply:
On second thought, I agree, it is not that special after all. We're just not at the point of self-hosting ILC yet, I suppose :).

Reply:
Yes, that' would be a project.  Although some parts of this, e.g. emscripten already run under Wasm.  A project for someone with more time on their hands than I have.